apps/docs/package.json

```diff
@@ -9,15 +9,27 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack",
     "package:serverless": "sls package",
+    "heroku-postbuild": "npm run build",
     "clean": "rm -rf .serverless dist",
     "dev": "NODE_ENV=development node-hot --fork | bunyan",
-    "dev:ssr": "NODE_ENV=development SSR_ONLY=true node-hot --fork | bunyan"
+    "dev:ssr": "NODE_ENV=development SSR_ONLY=true node-hot --fork | bunyan",
+    "create": "plop",
+    "create:deployment": "plop deployment",
+    "create:page": "plop page"
```

Comment:
Should the plops be included here or can they just run from the upstream project via `npx`?
Should they only be included here if the upstream defaults have been changed to do something project specific?